feature/gribi/otg_tests/gribi_static_route_on_lemmings_test/gribi_static_route_on_lemmings_test.go

```diff
@@ -0,0 +1,185 @@
+package gribi_static_route_on_lemmings_test
```

Comment:
probably we want to exclude lemming from the namings here ... 

the test seems to be agnostic of platform and the lemmings option is passed as a flag to the test in wbb bind -- use_kne_topo ...

I agree for the need of a simple test in case you want to demo.

Reply:
If I remove the lemmings from the name, it will look like a generic one and will be duplicated due to other test cases, which was covering all the aspects, also this coverage is required to have it for Lemming development. We can execute the test case only on Lemming by executing the below command 

blaze test --notest_loasd --test_output=streamed --cache_test_results=no -- //third_party/openconfig/featureprofiles/feature/gribi/otg_tests/gribi_static_route_on_lemmings_test:gribi_static_route_on_lemmings_test_kne_openconfig_lemming

Reply:
I suspect if we can include platform aspects into the test files... the tests are really platform agnostic....

Also i believe this is not lemmings only test ... the same test should be able to work on other vendors as well... Also, i do not think the other tests cover this simple case right? 

+ @dplore for any thoughts

Reply:
Approving as this is not a blocker to run tests... 

@dplore when you have a chance .... please let us know on your thoughts so that we can do it in a seperate change if any is needed.